lib/MetaCPAN/Document/Author/Set.pm

```diff
@@ -0,0 +1,112 @@
+package MetaCPAN::Document::Author::Set;
+
+use strict;
+use warnings;
+
+use Moose;
```

Comment:
MetaCPAN::Moose is not used anywhere in API (not even in its cpanfile)... we can try switching to it, but that should be a separate commit (I'll start playing with it soon).
These packages were simply broken off as is from the Document files.